.travis.yml

```diff
@@ -37,6 +37,12 @@ matrix:
     - php: 7.0
       env: PHPCS=1 DEFAULT=0
 
+    - php: 7.0
```

Comment:
It will differ in case that something in PHP changed because I use reflection of the current version.